### PR TITLE
First HTML and CSS project- Youtube clone

### PR DESCRIPTION
**Pull request by Temmarie**
## Youtube Clone Final Changes: 
In this we;
- made changes to the interactive bar
- corrected the issues shown by the HTML validator

### CODE REVIEW

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Congrats on getting this far @Temmarie and @chindrop! :rocket: 
You've done a nice job creating this, however, there are some things I must bring to your attention before you can get an approval! Please see my comments below:

#### Descriptive PR
You need a more descriptive PR as it is a hard requirement. Consider describing the work you have completed in the branch in as much detail as you can and even include a link to the live project to speed up your review!
screenshot

#### Descriptive Readme
Your Readme file will benefit from a work up. When you start to apply to jobs, potential employers will go through your Github account to see the way you work and may form an opinion of the kind of developer you are from what they see. As such, it is important to ensure that all your repositories look very neat and professional. That being said, please consider this [example](https://github.com/Oluwadamilareolusakin/show-junkie-frontend/blob/master/README.md) as what your Readme File should look like! :smile: 

#### Live link
Please remember to update the live link in your project to point to the actual website. My go to resource for making live links is [Raw Githack](https://raw.githack.com) Please check it out, it's free and easy to use :+1: 

#### HTML Validation
Please condsider parsing your HTML markup through this validator and correct all resulting errors. It is mandatory that your markup is free from semantic or syntax errors.
screenshot

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

File: index.html
Comment (nikasulo) [inline index.html]: Please remove these comments from your code as they prevented me from viewing the live version of your project!

### NOTE TO SUBSEQUENT TSE: Please review the live project for potential changes to be made as I couldn't due to these comments. Thanks!

**Review by nikasulo — CHANGES_REQUESTED** [review]
Please make necessary corrections and submit for another review! All the best and see you at the top! :medal_military:

**Comment by chindrop** [comment]
We have tried so many methods but we are having so many header problem check other people's code when we opened our project the next alignment on the page was different others see it different also

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi team!
### Nice job! WOW, project looks! :+1: 

### But your project needs a few changes before it can be approved.

#### Kindly recommended checking HTML markup with this validator https://validator.w3.org/#validate_by_upload.
screen
 * Now it shows ERRORs in your project
errors

#### Sidebar Recommended videos
screen



### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*

*Happy coding 😄*

**Comment by chindrop** [comment]
We have fixed all the issues in the code and  submitting for code review

**Comment by rOluochKe** [comment]
@chindrop and @Temmarie Good job for reaching this far. However, there are changes you are required to make.
- Make your navigation bar better, [refer from youtube](https://www.youtube.com/watch?v=fnkWUW2hG0c)
right-side navigation
- amend your sidebar to look something like a youtube [playlist](https://www.youtube.com/watch?v=fnkWUW2hG0c)
amend-sidebar
- Make the song title better to look like one in youtube
title
- Add user avatar, name of artist, subscribers, description
comment section
- Add comment section below user details
commenting

Make all the changes and resubmit for review. Good luck

**Comment by chindrop** [comment]
## Youtube Page Clone: Changes made to the search bar and comment section
In the we 
- Changes made to the header and search bar.
- Comment section corrected.
- Correction made to subscribe section

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Under Review by Oluwadamilare (Tolulope) Olusakin
Great work guys! I am currently writing your review and will be done in less than an hour, please check in a few minutes 😊.

**Review by nikasulo — CHANGES_REQUESTED** [review]
Really great work @Temmarie and @chindrop :rocket: 
Please consider my comments below and the changes I am proposing. Make appropriate corrections and submit for another review :+1: 

#### Descriptive PR
You need a more descriptive PR as it is a hard requirement. Consider describing the work you have completed in the branch in as much detail as you can and even include a link to the live project to speed up your review!
screenshot

#### HTML Validation
Please condsider parsing your HTML markup through this validator and correct all resulting errors. It is mandatory that your markup is free from semantic or syntax errors.
screenshot

#### No Stickler requirement
I really commend your efforts in attempting to setup stickler on your repository, while it is generally not required for this project, if you do want to use it please ensure you correctly set it up and fix all the errors it throws! :+1: 

#### Subscribe button
Please consider aligning the subscribe button to towards the top of the div. It is currently aligned in the middle which is not where it should be in comparison to what exists on the original site. Please make appropriate changes to this!
screenshot


##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Comment by Temmarie** [comment]
## Youtube Clone Final Changes: HTML validation and Subscribe button

In this milestone we:
- Validated the Index.html and made corrections
- Positioned the subscribe button to the right spot

**Comment by shshamim63** [comment]
## Status :ballot_box_with_check: [Changes Required] :ballot_box_with_check: 
___
Hi @Temmarie and @chindrop :wave: ,
You have covered most of the task asked by previous reviewer. You have gathered almost all the elements to decorate the page, and most of them are on the right place, great job :clap: . But there are some issues that are blocking me to approve this project. And I am pretty much sure that you will cover them too.
___
#### Please follow the instructions below to complete the project.
___
### :spades: [Mandatory Changes] :spades: 
___
1. Specification for the video-> 

- [ ] It has like and dislike icons ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_like_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
- [ ] It has a share button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))

Your have covered those thing by using a screenshot. Where they should be individual components.
image

2. Still I can see some error are yelling on HTML markup validator. you were suppose to check HTML markup with this validator https://validator.w3.org/#validate_by_upload.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from Pathwright). Please resolve them.

3. You were suppose to add a summary about the project in here
https://github.com/chindrop/First_Html-CSS_Project/pull/1#issue-363507843
But I can not see it. A summary on the task you have covered in this PR.
___
#### Please fix the above mentioned issue and send another review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/)

**Comment by Temmarie** [comment]
## Youtube Clone Final Changes: 
In this we;
- made changes to the interactive bar
- corrected the issues shown by the HTML validator

**Comment by shshamim63** [comment]
## Status :heavy_check_mark: Approved :heavy_check_mark: 
___
Hi @Temmarie  and @chindrop :wave: ,
Project contains a readme, code looks neat and clean. No markup validation error. Page contains all the items needed to decorate the page. Covered all the changes mentioned on the previous reviews. Just got some suggestion to make you page make more look alike the original one.
___
### :clubs: [Optional Changes] :clubs: 
___
1. There is too much gap between the header item and the main items.  If you decrease it a little bit, that will looks awesome.
image
___
### This :+1: is from me. You can merge it and don't forget to fill the project completion form.
___
## Shakhawat Hossain

**Pull request merged**
